tools/internal_ci/helper_scripts/prepare_build_windows.bat

```diff
@@ -14,7 +14,7 @@
 
 @rem make sure msys binaries are preferred over cygwin binaries
 @rem set path to python 2.7
-set PATH=C:\tools\msys64\usr\bin;C:\Python27;C:\Python37;%PATH%
+set PATH=C:\tools\msys64\usr\bin;C:\Python27;%PATH%
```

Comment:
This line differs from https://github.com/grpc/grpc/pull/16111?

Reply:
Closed the original in favor of getting this on master via the next upmerge to ensure they will be identical.